rviz_common/src/rviz_common/frame_manager.cpp

```diff
@@ -291,13 +291,6 @@ const std::string & FrameManager::getFixedFrame()
   return fixed_frame_;
 }
 
-std::string getTransformStatusName(const std::string & caller_id)
-{
-  std::stringstream ss;
-  ss << "Transform [sender=" << caller_id << "]";
-  return ss.str();
-}
-
 #if 0
```

Comment:
I'm also for removing this block, and basically any block that is `#if 0` out.  At this point in rviz2's life, if we need functionality like that we can:

1.  Get it from git history.
2.  Port it from ROS 1 rviz.
3.  Recreate it from scratch.